authorization-common/src/test/java/com/sequenceiq/authorization/utils/EventAuthorizationUtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.authorization.utils;
+
+import static java.util.Collections.emptySet;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import com.sequenceiq.authorization.resource.AlternativeAuthorizationResourceActionProvider;
+import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
+import com.sequenceiq.authorization.service.CommonPermissionCheckingUtils;
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
+
+class EventAuthorizationUtilsTest {
```

Comment:
Please add tests for all resource types that it checks for the expected permission.

Reply:
Will do, thanks

Reply:
nit: please be consistent and follow the pattern in the module: use visibility modifier for classes and module, public for test classes and test methods